lib/app/migrate.ts

```diff
@@ -414,25 +436,26 @@ export async function migrateApp2025_2(
     derivedAccountId,
     migrationId,
     uidMap,
-    projectName
+    projectConfig?.projectConfig?.name || projectName
   );
 
   await downloadProjectFiles(
     derivedAccountId,
     projectName,
     buildId,
-    projectDest
+    projectDest,
+    projectConfig
   );
 }
 
-export function logInvalidAccountError(i18nKey: string): void {
+export function logInvalidAccountError(): void {
   uiLine();
-  logger.error(i18n(`${i18nKey}.errors.invalidAccountTypeTitle`));
+  logger.error(lib.migrate.errors.invalidAccountTypeTitle);
   logger.log(
-    i18n(`${i18nKey}.errors.invalidAccountTypeDescription`, {
-      useCommand: uiCommandReference('hs accounts use'),
-      authCommand: uiCommandReference('hs auth'),
-    })
+    lib.migrate.errors.invalidAccountTypeDescription(
+      uiCommandReference('hs accounts use'),
```

Comment:
We prefer `hs account` singular now. Also perhaps promote `hs account auth` here as well? 😁

Reply:
I might just keep it as `hs account use` for now because I don't want a person to be mid migration and then onboard to the new config.  Might be nice to limit the moving pieces